Anlab.Mvc/Client/components/PaymentSelection.tsx

```diff
@@ -17,18 +18,36 @@ export class PaymentSelection extends React.Component<IPaymentProps, any> {
     constructor(props) {
         super(props);
         this.state = {
-            error: ""
+            error: "",
+            accountName: null
         };
 
 
     }
     _renderUcAccount = () => {
         if (this.props.payment.clientType === 'uc') {
             return (
-                <Input type="text" label="UC Account" error={this.state.error} value={this.props.payment.account} maxLength={15} onChange={this.handleAccountChange}/>
+                <div>
+                    <Input type="text" label="UC Account" error={this.state.error} value={this.props.payment.account} maxLength={15} onChange={this.handleAccountChange} onBlur={this.lookupAccount} />
+                    {this.state.accountName}
+                </div>
             );
         }
     }
+
+    lookupAccount = (account: string) => {
+        if (this.state.error === "")
```

Comment:
@laholstege it's all the same but since empty string is falsey you can just do `if (!this.state.error)` and it'll handle other cases like null and undefined